router/batchrouter/asyncdestinationmanager/marketo-bulk-upload/configParser.go

```diff
@@ -0,0 +1,26 @@
+package marketobulkupload
+
+import "encoding/json"
+
+func (m *MarketoConfig) UnmarshalJSON(data []byte) error {
+	var intermediate intermediateMarketoConfig
+	if err := json.Unmarshal(data, &intermediate); err != nil {
```

Comment:
lets use jsonfast here instead of json

Reply:
Updated in all relevant places where `encoding/json` was used